packages/jss-vendor-prefixer/src/index.test.js

```diff
@@ -0,0 +1,165 @@
+import expect from 'expect.js'
+import {create} from 'jss'
+import cssVendor from 'css-vendor'
+import browser from 'detect-browser'
+import functionPlugin from 'jss-plugin-syntax-rule-value-function'
+
+import vendorPrefixer from './index'
+
+const settings = {
+  createGenerateClassName: () => rule => `${rule.key}-id`
+}
+
+const isIE9 = browser.name === 'ie' && browser.version === '9.0.0'
+const isIEorEdge = browser.name === 'edge' || browser.name === 'ie'
+
+describe('jss-vendor-prefixer', () => {
+  let jss
+
+  beforeEach(() => {
+    jss = create(settings).use(vendorPrefixer())
+  })
+
+  describe('prefixed property', () => {
+    if (isIE9) {
+      return
+    }
+
+    let sheet
+
+    beforeEach(() => {
+      sheet = jss.createStyleSheet({
+        a: {animation: 'yyy'}
+      })
+    })
+
+    it('should generate correct CSS', () => {
+      if (isIEorEdge) {
+        expect(sheet.toString()).to.be('@keyframes a {\n}')
```

Comment:
Why will this be `keyframes` @kof? Doesn't make sense to me. This test fails in the BrowserStack tests but runs successfully locally.

Reply:
doesn't make sense to me, cc @AleshaOleg https://github.com/cssinjs/jss-vendor-prefixer/commit/739a34128c2d84395fc441093710bf1cedd4a9b8